nodejs-repl.el

```diff
@@ -135,7 +139,7 @@ See also `comint-process-echoes'"
 
 (defvar nodejs-repl-code-format
   (concat
-   "require('repl').start('%s', null, null, true, false, "
+   "require('repl').start({prompt: '%s', useGlobal: %s}, null, null, true, false, "
```

Comment:
`repl.start` has two signatures, one is `repl.start(options)` and another is `repl.start(prompt, stream, eval, useGlobal, ignoreUndefined, replMode)`. If you specify a object as the first argument, other arguments are ignored, so please change the code like below:

```diff
diff --git a/nodejs-repl.el b/nodejs-repl.el
index 58834bb..32f758c 100644
--- a/nodejs-repl.el
+++ b/nodejs-repl.el
@@ -139,8 +139,8 @@ See also `comint-process-echoes'"

 (defvar nodejs-repl-code-format
   (concat
-   "require('repl').start({prompt: '%s', useGlobal: %s}, null, null, true, false, "
-   "require('repl')['REPL_MODE_' + '%s'.toUpperCase()])"))
+   "require('repl').start({ prompt: '%s', useGlobal: %s, replMode: "
+   "require('repl')['REPL_MODE_' + '%s'.toUpperCase()] })"))

 (defvar nodejs-repl-extra-espace-sequence-re "\\(\x1b\\[[0-9]+[GJK]\\)")

```